content/docs/contributing/to-docs/blog-posts.md

```diff
@@ -0,0 +1,106 @@
+
+9. Go into the O3DE Discord server and share your pull request for review. (Alternatively, mail the sig-docs-community mailing list with a link.) Someone will review it and provide feedback or an okay. A maintainer will merge it when at least one reviewer approves it.
+
+{{< note >}}
+The O3DE Community reserves the right to not publish any blog post that does not meet our standards. We also reserve the right to remove any currently published blog post that might conflict with the agreed-upon O3DE strategy or delivery.
+{{< /note >}}
+
+## What makes a good O3DE blog post?
+
+Good question! That's really up to the community alongside our established Code of Conduct and Community Tenets. Not sure? Join the O3DE Discord or one of the SIGs, and vet your idea with our members.
+
+Here's some basic Do's and Don'ts to help you scope your effort and avoid an obvious rejection.
+
+**Do:**
+
+* Discuss O3DE! (Duh.) Go deep! Highly technical blog posts, in particular, establish community credibility and provide a resource for people looking for deeper knowledge beyond the tech docs.
+
+* Discuss algorithms, techniques, technologies, and tools that apply directly to O3DE work. Again, not only is your knowledge and experience useful to capture as a published topic, but it helps the larger industry assess the caliber of the people building O3DE.
+
+* O3DE strategy, **if** you are actively participating in a SIG around feature or product strategy, **and** have the approval of a SIG.
+
+* Interviews with, or profiles of, important and established community members. People put a human face on a technical product, and the world reliably likes to hear humans say they have to say. In this case, it should be professional observations, knowledge, and commentary&mdash;and shuld be specifically related to O3DE. Personal opinions, as ever, should be largely technical.
+
+* Write with strong grammar and an eye for easy readability. Rambling monologues may get a gentle request for editorial review and a number of suggestions to "tighten it up". The reader's time is precious, so compressing your thoughts into short, information-dense sentences with regular paragraph breaks is just respectful.
+
+* Include images and diagrams wherever appropriate.
+
+* Consider your audience, which is diverse across many sociocultural and industry axes. Choose your words with respect and care, and be prepared to take feedback sincerely and make changes. No-one knows everything about the folks in the community and the industry, so don't let some initially careless wording caught in review bring you down; just make the changes, internalize the feedback, and keep delivering great content!
+
+**Don't:**
+
+* Get personal. We have no tolerance for using the O3DE blog to call out community individuals&mdash;or any individuals, really&mdash;in any negative way, no matter how concerned you are.
```

Comment:
```suggestion
* Get personal. We have no tolerance for using the O3DE blog to call out community individuals&mdash;or any individuals, really&mdash;in any negative way, no matter how concerned you are.
```
Suggested rewrite:  Get personal. Do not use the O3DE blog to call out any individuals in a negative way.